python_modules/libraries/dagster-dg-cli/dagster_dg_cli/cli/plus/API_IMPLEMENTATION_PLAN.md

```diff
@@ -0,0 +1,413 @@
+
+### 14. **backfill**
+
+```bash
+dg plus api backfill list [--status <status>] [--json]
+dg plus api backfill view <backfill-id> [--json]
+dg plus api backfill create --asset <asset-key> --partitions <range>
+dg plus api backfill cancel <backfill-id>
+```
+
+**GraphQL**: `partitionBackfillsOrError`, `partitionBackfillOrError`
+
+### 15. **env-var**
+
+```bash
+dg plus api env-var list [--location <name>] [--json]
+dg plus api env-var view <var-name> [--location <name>] [--json]
+# Environment variables managed through deployment settings
+```
+
+**GraphQL**: `utilizedEnvVarsOrError`
+
+---
+
+## **Tier 4: Advanced/Monitoring**
+
+### 16. **alert-notification**
+
+```bash
+dg plus api alert-notification list [--policy-id <id>] [--limit <n>] [--json]
+dg plus api alert-notification view <notification-id> [--json]
+# Notifications are read-only audit trail
+```
+
+**GraphQL**: `alertPolicyNotifications`
+
+### 17. **audit-log**
+
+```bash
+dg plus api audit-log list [--user <user-id>] [--action <action>] [--limit <n>] [--json]
+dg plus api audit-log view <log-id> [--json]
+# Audit logs are read-only
+```
+
+**GraphQL**: `auditLog`
+
+### 18. **custom-role**
+
+```bash
+dg plus api custom-role list [--json]
+dg plus api custom-role view <role-id> [--json]
+dg plus api custom-role create --name <name> --permissions <file>
+dg plus api custom-role update <role-id> --permissions <file>
+dg plus api custom-role delete <role-id>
+```
+
+**GraphQL**: `customRoles`, `customRoleOrError`
+
+### 19. **api-token**
+
+```bash
+dg plus api api-token list [--type <user|agent>] [--json]
+dg plus api api-token view <token-id> [--json]
+dg plus api api-token create --description <desc> [--type <type>]
+dg plus api api-token revoke <token-id>
+```
+
+**GraphQL**: `apiTokensOrError`, `agentTokensOrError`
+
+### 20. **user-token**
+
+```bash
+dg plus api user-token list --user <user-id> [--json]
+dg plus api user-token view <token-id> [--json]
+dg plus api user-token create --user <user-id> --description <desc>
+dg plus api user-token revoke <token-id>
+```
+
+**GraphQL**: `userTokensOrError`
+
+---
+
+## **Additional Requested Nouns**
+
+### 21. **branch-deployment**
+
+```bash
+dg plus api branch-deployment list [--pr-status <open|merged>] [--json]
+dg plus api branch-deployment view <deployment-name> [--json]
+dg plus api branch-deployment create --branch <branch> --repo <repo>
+dg plus api branch-deployment delete <deployment-name>
+```
+
+**GraphQL**: `branchDeployments`
+
+### 22. **location-schema**
+
+```bash
+dg plus api location-schema list [--json]
+dg plus api location-schema view [--json]
+# Schema is typically read-only metadata
+```
+
+**GraphQL**: `locationSchema`
+
+### 23. **deployment-setting**
+
+```bash
+dg plus api deployment-setting list [--json]
+dg plus api deployment-setting view [--json]
+dg plus api deployment-setting update --config <file>
+# Settings are typically singular per deployment
+```
```

Comment:
organization-setting would be good too (esp since that is the only way to set organization settings currently)